src/app/business/profile/profileCard/profile-card.component.ts

```diff
@@ -104,12 +104,7 @@ export class ProfileCardComponent implements OnInit {
         private confirmationService:ConfirmationService
     ) { }
     option = {};
-    pools = [];
-    totalFreeCapacity = 0;
-    totalCapacity = 0;
     ngOnInit() {
-        this.getProfiles();
```

Comment:
any specific reason we do this on init ?

Reply:
@rhsakarpos The code that is marked here is removed. It used to be called at the time of init and that was one of the reasons for multiple GET calls for profiles. This has now been removed.

Reply:
LG...AB